src/DiagramGenerator/Diagram/Board.php

```diff
@@ -216,7 +209,7 @@ public function drawBorder()
      */
     public function draw()
     {
-        $this->image->setImageFormat('png');
+        $this->image->setImageFormat(Texture::IMAGE_FORMAT_PNG);
```

Comment:
Why? The point of this was to output JPEG files when the source image was JPEG, and PNG when it was PNG. Does this force everything to output as PNG?